blazor/file-manager/how-to/add-custom-tool-bar.md

```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Adding Custom Item to Toolbar in Blazor FileManager Component
 
-The toolbar items can be customized using the `ToolbarSettings` API and `ToolbarItemClicked` events.
+The toolbar items can be customized using the `ToolbarSettings` API and `ToolbarCreated` events.
```

Comment:
The custom toolbar items can be added and customized using the `ToolbarSettings` API and `ToolbarCreated` events.

Reply:
Content modified @rajendranr-5483  sir